arklib/Cargo.toml

```diff
@@ -15,8 +15,8 @@ url = { version = "2.2.2", features = ["serde"] }
 openssl = { version = "0.10.42", features = ["vendored"] }
 image = "0.24.2"
 log = "0.4.14"
-
-[target.'cfg(target_os="android")'.dependencies]
 jni = { version = "0.19.0", default-features = false }
 android_logger = "0.11.0"
+
+[target.'cfg(target_os="android")'.dependencies]
 ndk = "0.6.0"
```

Comment:
Am I right that NDK is useless on desktop?

Reply:
Yes, moreover there will be an error
` compile_error!("android-ndk-sys only supports compiling for Android");`